### ISSUE TITLE
No auth dialog for openvpn

### ISSUE THREAD

**Issue by baude** [decathorpe/elementary-stable-rpms#14]
When trying to use my openvpn with pantheon, I do not get the password prompt.  I think there is a conflict between the gnome and pantheon polkit stuffs?

**Comment by decathorpe** [comment]
There shouldn't be, since the Pantheon polkit agent only runs in a Pantheon session, and the gnome polkit agent only runs in gnome sessions.

Did you set up the VPN system-wide (for all users)? Otherwise polkit shouldn't have anything to do with this problem at all, I think

**Comment by baude** [comment]
It is setup for all users ... are you suggesting I try it without it being setup that way?

**Comment by decathorpe** [comment]
Have you checked if the `pantheon-agent-polkit` package is installed? You might have to install it manually.

If it is there's a different problem, but setting up the VPN for your user only might work around the problem.

**Comment by baude** [comment]
Yeah I had that polkit package installed and there was an error in the logs suggesting some conflicts. When work is done, I'll switch over to Pantheon and try again and capture more information.

**Comment by baude** [comment]
Out of journalctl I can see:

``
`Failed to request VPN secrets #3: No agents were available for this request.
```
It looks like people have observed this problem in F26.  It works if I call the VPN through nmcli but not through the GUI.

My pantheon related RPMS are:
```
 2017-10-18 19:05:49 ⌚  localhost in ~
○ → rpm -qa | grep -i pantheon
pantheon-desktop-0.1.1-1.fc26.noarch
pantheon-files-0.3.5-1.fc26.x86_64
pantheon-online-accounts-0.3.0.1-1.20170417.git5a0270a.fc26.x86_64
pantheon-mail-1.0.6-1.fc26.x86_64
switchboard-plug-pantheon-shell-0.2.6-1.fc26.x86_64
pantheon-session-0.1-1.fc26.noarch
pantheon-calculator-0.1.3-1.fc26.x86_64
pantheon-session-settings-wayland-0.9.90-1.fc26.noarch
pantheon-online-accounts-libs-0.3.0.1-1.20170417.git5a0270a.fc26.x86_64
pantheon-photos-0.2.4-1.fc26.x86_64
pantheon-terminal-0.4.3-1.fc26.x86_64
pantheon-session-settings-0.9.90-1.fc26.noarch
pantheon-agent-polkit-0.1.4-1.fc26.x86_64
```

**Comment by decathorpe** [comment]
That's strange. Is the pantheon polkit agent even running in your Pantheon session? The binary is called `pantheon-agent-polkit`, located at `/usr/libexec/policykit-1-pantheon/pantheon-agent-polkit`

**Comment by baude** [comment]
Looks like it is running to me.
```
± |master → origin U:2 ?:2 ✗| → ps aux | grep agent
bbaude    2665  0.0  0.0  70144   788 ?        Ss   18:28   0:00 /usr/bin/ssh-agent /bin/sh -c exec -l /bin/bash -c "gnome-session --session=pantheon"
bbaude    3002  0.0  0.0 538880 20784 tty2     Sl+  18:28   0:00 /usr/libexec/policykit-1-pantheon/pantheon-agent-polkit
```

**Comment by decathorpe** [comment]
Ah, I think I know what's going on.

Is the seahorse daemon running? That's the one that provides saved credentials ...

**Comment by baude** [comment]
What package provides it ?

**Comment by baude** [comment]
I dont see it:

```
 2017-10-22 09:00:24 ⌚  localhost in ~
○ → ps aux | grep sea
gdm       3155  0.0  0.0 1469208 10952 ?       S<l  08:56   0:00 /usr/bin/pulseaudio --start --log-target=syslog
bbaude    3870  0.2  0.0 2026940 12804 ?       S<l  08:59   0:00 /usr/bin/pulseaudio --start --log-target=syslog
bbaude    4087  0.0  0.0 309000 17496 tty2     S+   08:59   0:00 /usr/bin/seapplet
```

**Comment by decathorpe** [comment]
I'll have to do some more digging into what's going wrong here exactly ...
